lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -694,6 +702,7 @@ class AzureRMVirtualMachine(AzureRMModuleBase):
     def __init__(self):
 
         self.module_arg_spec = dict(
+            availability_zone=dict(type='int', choices=[1, 2, 3]),
```

Comment:
should we accept list instead of a single one?

Reply:
From my understanding, a virtual machine can be only created in one availability zone. From the user's perspective having to provide a list with a single element can be a bit misleading I think.